bootstrap/bootstrap.go

```diff
@@ -30,6 +31,8 @@ func Start(ctx context.Context, cfg *config.Config) error {
 	logger = logger.Level(cfg.LogLevel)
 	logger.Info().Msg("starting up the EVM gateway")
 
+	metricsServer := metrics.NewServer(logger, 9091)
```

Comment:
I'm leaving it 9091 for now. Do you want me to parse it from `prometheus.yml` for consistency?

Reply:
best would be to parse the yaml